io.github.kitware.F3D.metainfo.xml

```diff
@@ -0,0 +1,68 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop">
+    <id>io.github.kitware.F3D</id>
```

Comment:
is this in fact the best app-id? kitware.com exists and that's where they host the sourcecode. Does the app make use of an app-id internally currently?

Reply:
I don't find any file matching appdata or metadata, so I don't think there is.
They use gitlab to host source code, but it seems like they use github as there home page website, I will ask them to clarify on this.

Reply:
Yeah, `com.kitware.F3D` may be better here.